tests/e2e/helpers/api/api-database.ts

```diff
@@ -17,8 +16,20 @@ export async function addNewStandaloneDatabase(databaseParameters: AddNewDatabas
         .send({ "name": databaseParameters.databaseName, "host": databaseParameters.host, "port": databaseParameters.port })
         .set('Accept', 'application/json');
 
-    await t.expect(await response.status).eql(201);
-    await t.expect(await response.body.name).eql(databaseParameters.databaseName);
+    await t.expect(await response.status).eql(201, 'addNewStandaloneDatabase request failed');
```

Comment:
Instead of `addNewStandaloneDatabase request failed` please write something such as `The creation of standalone database`
Same bellow. Do not use synonymous such `addNewStandaloneDatabase` in the error message. It should be readable for anyone.

Reply:
Thanks, fixed